cmd/ib-sriov-cni/main.go

```diff
@@ -39,14 +39,18 @@ func init() {
 }
 
 func getGUIDFromConf(netConf *localtypes.NetConf) (string, error) {
+	// Take from CNI_ARGS if available
+	if netConf.CniArgGUID {
+		if guid, ok := netConf.Args.CNI["guid"]; ok {
```

Comment:
so you need to move the RuntimeConfig.InfinibandGUID  to here as well (Line 50). We will use this once InfinibandGUID   will be approved by the network-att-def same as mac see [1]

[1] https://docs.google.com/document/d/1V0R_MEiPapZOD_SuBjzTOdGYqILBFovuy1xacYCqFHg/edit

Reply:
Done.
I thought with ib-kubernetes, it only allocates GUID for VF device, but it actually respect what's allocated in CNI_ARGS["guid"].